docker/run_local_docker.sh

```diff
@@ -87,7 +87,7 @@ docker compose  -f docker/local-metadata/docker-compose-postgres.yml down
 docker compose -f docker/local-metadata/docker-compose.yml down
 
 echo "Starting Local Docker Containers"
-mkdir -p docker-volume && mkdir -p docker-volume/db-data
+mkdir -p docker-volume && mkdir -p docker-volume/db-data && mkdir -p docker-volume/es-data
```

Comment:
This is not required as the docker-volume folder is created in the directory where docker-compose file is! In case of local docker run, it's docker/local-metadata directory. So let's remove this code from here.

Reply:
Done